crates/typst/src/eval/str.rs

```diff
@@ -346,6 +346,27 @@ impl Str {
         self.locate_opt(index)?
             .ok_or_else(|| out_of_bounds(index, self.len()))
     }
+
+    // Capitalize the first letter of every word.
```

Comment:
These should be doc-comments (three leading slashes) and be in third person: `/// Capitalizes the first letter of every word.`